src/styles/problem.scss

```diff
@@ -28,7 +28,7 @@ td.problem-cell {
 .contest-statistics {
     position: absolute;
     bottom: 10px;
-    left: 16px;
+    left: -16px;
```

Comment:
It will also affect LeetCode's style, let's overwrite in &.code-chef.

<img width="266" alt="image" src="https://github.com/user-attachments/assets/b98fde18-63f1-4b50-83fd-290301127148">

Reply:
made necessary changes.